kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/main/java/org/kie/server/remote/rest/jbpm/ui/ImageResource.java

```diff
@@ -93,13 +99,18 @@ public Response getProcessImage(@javax.ws.rs.core.Context HttpHeaders headers,
     @Path(PROCESS_INST_IMG_GET_URI)
     @Produces({MediaType.APPLICATION_SVG_XML})
     public Response getProcessInstanceImage(@javax.ws.rs.core.Context HttpHeaders headers,
-            @ApiParam(value = "container id that process instance belongs to", required = true, example = "evaluation_1.0.0-SNAPSHOT") @PathParam(CONTAINER_ID) String containerId, 
-            @ApiParam(value = "identifier of the process instance that image should be loaded for", required = true, example = "123") @PathParam(PROCESS_INST_ID) Long procInstId) {
+                                            @ApiParam(value = "container id that process instance belongs to", required = true, example = "evaluation_1.0.0-SNAPSHOT") @PathParam(CONTAINER_ID) String containerId,
+                                            @ApiParam(value = "identifier of the process instance that image should be loaded for", required = true, example = "123") @PathParam(PROCESS_INST_ID) Long procInstId,
+                                            @ApiParam(value = "svg completed node color", required = false, example = COMPLETED_COLOR) @QueryParam(SVG_NODE_COMPLETED_COLOR) @DefaultValue(COMPLETED_COLOR) String svgNodeCompletedColor,
+                                            @ApiParam(value = "svg completed node border color", required = false, example = COMPLETED_BORDER_COLOR) @QueryParam(SVG_NODE_COMPLETED_BORDER_COLOR) @DefaultValue(COMPLETED_BORDER_COLOR) String svgNodeCompletedBorderColor,
+                                            @ApiParam(value = "svg active node color", required = false, example = ACTIVE_COLOR) @QueryParam(SVG_NODE_ACTIVE_COLOR) @DefaultValue(ACTIVE_COLOR) String svgNodeActiveColor) {
         Variant v = getVariant(headers);
         Header conversationIdHeader = buildConversationIdHeader(containerId, context, headers);
         try {
-
-            String svgString = imageServiceBase.getActiveProcessImage(containerId, procInstId);
+             String svgString = imageServiceBase.getActiveProcessImage(containerId, procInstId,
```

Comment:
just check the formatting here that is a bit off.

Reply:
yes, fixed